json/Item_Weapon_Partizan.txt

```diff
@@ -907,7 +907,7 @@
 		"jp_text": "氷柱華",
 		"tr_text": "Tsurara Hana",
 		"jp_explain": "刃に冷気を放つ鉱石を用いた長槍。\n美しい刃に見惚れ、触れようとした者は\n皆冷気によって氷漬けとなる。",
-		"tr_explain": ""
+		"tr_explain": "A Partizan with a crystalline blade.\nEmitting an aura of cold, it freezes\nfoes solid with its mere touch."
```

Comment:
Item_Weapon_Partizan.txt: Invalid control character '\n' at: line 910 column 125 (char 37105)
Invalid string: control characters from U+0000 through U+001F must be escaped at line 911, column 1